### PR TITLE
Show changed files in overview

### LINKED ISSUE #350: Provide a summary list of changed files/folders in an activity

### PR DESCRIPTION
**Pull request by tmaybe**
Calculates and displays a summary of the changed files in an activity at the top of the Activity Overview page.

Closes #350

### CODE REVIEW

**Comment by tmaybe** [comment]
Note that the summary page will now crash when trying to render activities that were started with versions of Chime previous to commit 44b756e88aee405335930ac91d0b9bdccbd84104

**Comment by tmaybe** [comment]
Not ready to merge yet, found a bug I'm chasing down.

**Comment by tmaybe** [comment]
This is now ready to merge, @wpietri or @migurski !

File: chime/view_functions.py [hunk 1]
```diff
@@ -588,6 +587,92 @@ def make_activity_history(repo):
...
+        except:
+            # we weren't give what we need, return an empty summary
+            return summary
+
+    ed_lookup = {'create': u'created', 'edit': u'edited', 'delete': u'deleted'}
+    change_lookup = {}
+    display_types_encountered = []
+    for action in reversed(history):
+        # we only care about edits
+        if action['commit_category'] == u'edit':
```
Comment (migurski) [inline chime/view_functions.py]: This is essentially a filter; can the stuff in the `if`-clause be moved up a level, and prefixed with something like this?

    history = [action for action in reversed(history) if action['commit_category'] == u'edit'`

**Comment by migurski** [comment]
Other than my one comment it looks pretty good.

**Comment by migurski** [comment]
I like the improvement in clarity from removing a conditional and one level of indentation! :+1:

**Pull request merged**